hack/multi-node/user-data.sample

```diff
@@ -19,36 +19,58 @@ coreos:
           [Unit]
           Requires=flanneld.service
           After=flanneld.service
+    - name: kubelet-initial-config.service
+      enable: true
+      command: start
+      content: |
+        [Service]
+        ExecStart=/home/core/install-initial-kubelet-config
+
+        [Install]
+        WantedBy=multi-user.target
     - name: kubelet.service
       enable: true
       command: start
       content: |
+        [Unit]
+        After=kubelet-initial-config.service
+
         [Service]
         EnvironmentFile=/etc/environment
+        EnvironmentFile=/etc/kubernetes/kubelet.env
         Environment=KUBELET_ACI=quay.io/coreos/hyperkube
-        Environment=KUBELET_VERSION=v1.3.6_coreos.0
         ExecStartPre=/bin/mkdir -p /etc/kubernetes/manifests
         ExecStartPre=/bin/mkdir -p /srv/kubernetes/manifests
         ExecStartPre=/bin/mkdir -p /etc/kubernetes/checkpoint-secrets
         ExecStart=/usr/lib/coreos/kubelet-wrapper \
           --api-servers=https://172.17.4.101:443 \
           --kubeconfig=/etc/kubernetes/kubeconfig \
           --lock-file=/var/run/lock/kubelet.lock \
-          --exit-on-lock-contention \
-          --config=/etc/kubernetes/manifests \
           --allow-privileged \
-          --hostname-override=${COREOS_PUBLIC_IPV4} \
-          --node-labels=master=true \
+          --config=/etc/kubernetes/manifests \
           --minimum-container-ttl-duration=3m0s \
           --cluster_dns=10.3.0.10 \
-          --cluster_domain=cluster.local
+          --cluster_domain=cluster.local \
+          --exit-on-lock-contention=true \
+          --node-labels=master=true,update-controller-managed=true \
+          --hostname-override=$public_ipv4
```

Comment:
Unless there is a specific reason we want the change, can we switch it back - this is specific to certain cloud-providers. The COREOS_PUBLIC_IPV4 is just coming from /etc/environment file -- which is often pre-populated but would also allow a deployer to easily populate this info.